test-case/check-kmod-load-unload.sh

```diff
@@ -97,15 +97,15 @@ do
         exit 1
     fi
 
-    dlogi "checking for fw_boot success"
-    keyword_info=$(dmesg |grep sof-audio | grep 'boot complete')
+    dlogi "checking if firmware boot successfully"
+    keyword_info=$(dmesg |grep sof-audio | grep 'Firmware debug build')
```

Comment:
I would suggest to use `Firmware info`
```suggestion
    keyword_info=$(dmesg |grep sof-audio | grep 'Firmware info')
```
`Firmware debug build` may not appear if FW is not debug build version.

Reply:
No, It will always appear. There are still some error checks after 'Firmware info', so it is not as robust as using 'Firmware debug build'. Though there is a 'debug' in the key word, but it has nothing to do with debug. And I have checked other dev_info logs, 'Firmware debug build' is the best.
```
	if (ready->flags & SOF_IPC_INFO_BUILD) {
		dev_info(sdev->dev,
			 "Firmware debug build %d on %s-%s - options:\n"
			 " GDB: %s\n"
			 " lock debug: %s\n"
			 " lock vdebug: %s\n",
			 v->build, v->date, v->time,
			 (ready->flags & SOF_IPC_INFO_GDB) ?
				"enabled" : "disabled",
			 (ready->flags & SOF_IPC_INFO_LOCKS) ?
				"enabled" : "disabled",
			 (ready->flags & SOF_IPC_INFO_LOCKSV) ?
				"enabled" : "disabled");
	}
```